### ISSUE TITLE
Null pointer - EntityCombustByEntityEvent to Essentials v2.17.2.114

### ISSUE THREAD

**Issue by sammyt291** [EssentialsX/Essentials#3369]
### Information

**Full output of `/ess version`**:

```
[01:27:10 INFO]: Server version: 1.15.2-R0.1-SNAPSHOT git-Paper-334 (MC: 1.15.2)
[01:27:10 INFO]: EssentialsX version: 2.17.2.114
[01:27:10 INFO]: GroupManager version: 0.3.0
[01:27:10 INFO]: PlaceholderAPI version: 2.10.4
[01:27:10 INFO]: Vault version: 1.7.2-b107
[01:27:10 INFO]: Citizens version: 2.0.26-SNAPSHOT (build 1951)
[01:27:10 INFO]: EssentialsXProtect version: 2.17.2.114
[01:27:10 INFO]: EssentialsXChat version: 2.17.2.114
[01:27:10 INFO]: EssentialsXSpawn version: 2.17.2.114
[01:27:10 INFO]: You are running unsupported plugins!
```

**Server log**: https://gist.github.com/sammyt291/22ce1f029a83c62c32dc185e4c4a0454

**EssentialsX config**: https://gist.github.com/sammyt291/a05ffd9da18132dd38c74f07c5e526a0

### Details

**Description**  
Unsure of an exact cause, most likely reason is from NPCs being killed by other NPCs in one hit with a flame bow.

**Steps to reproduce**  
Have a citizens-sentinels NPC kill another NPC with a flame bow indirectly.?
Unsure,

**Expected behavior**  
No error

**Screenshots**  
N/a

**Comment by triagonal** [comment]
Similar to issue #1869, which was thought to have been fixed.

**Comment by JRoy** [comment]
Null check here would prob fix this? 
https://github.com/EssentialsX/Essentials/blob/46ed8e954f091c0f92f075862cc741f477c0bd4e/Essentials/src/com/earth2me/essentials/EssentialsEntityListener.java#L131

**Comment by triagonal** [comment]
This has been fixed in EssentialsX versions `2.17.2.141` and above. Download the latest build [here](https://essentialsx.net/downloads.html).
